MonitoringTools/usr/local/bin/send_metric_utils.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict, Tuple, Callable, List
+from pathlib import Path
+import configparser
+import requests
+import argparse
+from configparser import ConfigParser
+
+
+def read_config_file(config_filepath: Path) -> Dict:
+    """
+    This function reads a config file and puts it into a dictionary
+    :param config_filepath:
+    :return: A flattened dictionary containing key-value pairs from config file
+    """
+    config = ConfigParser()
+    config.read(config_filepath)
+    config_dict = {}
+    for section in config.sections():
+        for key, value in config.items(section):
+            config_dict[f"{section}.{key}"] = value
+
+    required_values = [
+        "auth.password",
+        "auth.username",
+        "cloud.instance",
+        "db.database",
+        "db.host",
+    ]
+    assert all(
+        val in config_dict for val in required_values
+    ), "Config file is missing required values."
+    return config_dict
+
+
+def post_to_influxdb(
+    data_string: str, host: str, db_name: str, auth: Tuple[str, str]
+) -> None:
+    """
+    This function posts information to influxdb
+    :param data_string: data to write
+    :param host: hostname and port where influxdb can be accessed
+    :param db_name: database name to write to
+    :param auth: tuple of (username, password) to authenticate with influxdb
+    """
+    if not data_string:
+        return
+
+    url = f"http://{host}/write?db={db_name}&precision=s"
+    response = requests.post(url, data=data_string, auth=auth)
+    response.raise_for_status()
+
+
+def parse_args(inp_args, description: str = "scrape metrics script") -> Dict:
```

Comment:
IMO I'd just hardcode the path to somewhere in /etc/

Reply:
I'd rather not hardcode.
I'd much rather have paths set and changable in one place - i.e. in Aquilon

Reply:
This goes double if we want to package-ify it